### PR TITLE
[BACKLOG-35528] convert log4j.xml to log4j2.xml with file rename

### PR DESCRIPTION
**Pull request by ddiroma**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 30m 22s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
assemblies/pentaho-solutions,extensions,repository
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by rmansoor — APPROVED** [review]

File: assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/osgi/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                   filePattern="../logs/osgi_pentaho.%d{yyyy-MM-dd}.log">
+         <PatternLayout>
+            <Pattern>%d %-5p [%c] %m%n</Pattern>
+         </PatternLayout>
+         <Policies>
+            <TimeBasedTriggeringPolicy />
+         </Policies>
+         <DefaultRolloverStrategy />
+      </RollingFile>
+      <Console name="PENTAHOCONSOLE">
```
Comment (bennychow) [inline assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/osgi/log4j2.xml:14]: This is missing the ThresholdFilter  set to ERROR.  The old log4j.xml had this in there.

File: extensions/src/main/resources/log4j/log4j2.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                   filePattern="../logs/pentaho.%d{yyyy-MM-dd}.log">
+         <PatternLayout>
+            <Pattern>%d %-5p [%c] %m%n</Pattern>
+         </PatternLayout>
+         <Policies>
+            <TimeBasedTriggeringPolicy />
+         </Policies>
+         <DefaultRolloverStrategy />
+      </RollingFile>
+      <Console name="PENTAHOCONSOLE">
```
Comment (bennychow) [inline extensions/src/main/resources/log4j/log4j2.xml:14]: Looks like this one had a previous threshold filter set to WARN.

**Review by bennychow — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 28m 37s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
assemblies/pentaho-solutions,extensions,repository
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by bennychow — APPROVED** [review]

**Pull request merged**